src/main/java/com/iota/iri/IRI.java

```diff
@@ -361,9 +118,49 @@ private static void shutdownHook() {
                 }
             }, "Shutdown Hook"));
         }
-    }
 
-    public static boolean validateParams(Configuration configuration, String[] args) throws IOException {
-        return IRILauncher.isValidated(configuration, args);
+        private static IotaConfig createConfiguration(String[] args) {
+            IotaConfig iotaConfig = null;
+            boolean testnet = ArrayUtils.contains(args, Config.TESTNET_FLAG);
+            File configFile = chooseConfigFile(args);
+            String message = "Configuration is created using ";
+
+            try {
+                if (configFile.exists()) {
```

Comment:
this fails quietly - for example:
```
$ java -jar iri-1.5.3.jar -c /bad/path/
Logging - property 'logging-level' set to: [INFO]
08/15 19:18:08.357 [main] INFO  com.iota.iri.IRI$IRILauncher - Configuration is created using command line args only
08/15 19:18:08.359 [main] INFO  com.iota.iri.IRI$IRILauncher - parsed the following cmd args: [-c, /bad/path/]
08/15 19:18:08.359 [main] INFO  com.iota.iri.IRI$IRILauncher - Welcome to IRI 1.5.3
...
```
so the file would not be loaded, and no message would be presented. (except "command line args only").